docs/en/your-profile-and-preferences/customize-direct-message-notifications.md

```diff
@@ -0,0 +1,85 @@
+---
+id: 597
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Customize direct message notifications
+
+By default, you receive notifications for direct messages (DMs). You can customize the notifications you receive in DMs. By doing so, you only receive the DMs notifications relevant to you.
+
+In addition to DMs you receive from others, you also receive notifications for mentions. To customize your mentions notifications, read [Customize mentions and other messages][customize-mentions-and-other-messages].
```

Comment:
'In addition to DM notifications, you also receive notifications for mentions.'

'To customize your **mention** notifications, **check out** <...>'